src/modules/launcher/Wox.Infrastructure/Image/WindowsThumbnailProvider.cs

```diff
@@ -125,5 +129,67 @@ private static IntPtr GetHBitmap(string fileName, int width, int height, Thumbna
 
             throw new InvalidComObjectException($"Error while extracting thumbnail for {fileName}", Marshal.GetExceptionForHR((int)hr));
         }
+
+        private static bool reportedAdobeReaderDetected = false; // Keep track if Adobe Reader detection has been logged yet.
```

Comment:
Do we need te reset it if Adobe Reader is not used? Then we can write a log if someone switches back to adobe.

Example: Adobe -> Test Foxit -> Back to Adobe.